impeller/renderer/stroke.comp

```diff
@@ -21,10 +21,18 @@ layout(binding = 1) buffer VertexBuffer {
 }
 vertex_buffer;
 
-layout(binding = 2) buffer VertexBufferCount {
-  uint count;
+// layout(binding = 2) buffer VertexBufferCount {
+//   uint count;
+// }
+// vertex_buffer_count;
+
+layout(binding = 2) buffer IndirectCommandArguments {
```

Comment:
This is the magic, a buffer with a specific struct layout (From https://developer.apple.com/documentation/metal/mtldrawprimitivesindirectarguments?language=objc) Can be supplied to a drawPrimitives call